e2e/capl-cluster-flavors/kubeadm-full-capl-cluster/chainsaw-test.yaml

```diff
@@ -214,6 +214,56 @@ spec:
             check:
               ($error == null): true
 
+    # Test to check if disks are added to the etcd spec
+    - clusters:
+        kubeadm-full-cluster:
+          kubeconfig: ./kubeadm-full-cluster-kubeconfig.yaml
+      name: Testing to check if disks are added to the spec
```

Comment:
we also want to check the disk creation from a pod by using exec. This will confirm that kubernetes is also able to see this new etcd-disk and its just not available from the linode side.